src/main/java/us/ilite/robot/RobotContainer.java

```diff
@@ -0,0 +1,82 @@
+package us.ilite.robot;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.RamseteController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryGenerator;
+import edu.wpi.first.math.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import us.ilite.common.config.Settings;
+import us.ilite.robot.modules.VioletDriveModule;
+
+import java.util.List;
+
+public class RobotContainer {
+    private final VioletDriveModule mDrive;
+    public RobotContainer(VioletDriveModule pModule) {
```

Comment:
Instead of making a whole new class, can we turn this into a Utility container. Basically instead of having to create yet another object, let's just make a factory method: 

```
public static Command buildTrajectoryCommand(VioletDriveModule pModule) {

}
```